src/font-weight-translations.tsx

```diff
@@ -1,8 +1,8 @@
-interface Map {
-  [key: string]: string | undefined;
-}
+import type { TextStyle } from 'react-native';
 
-const FontWeightTranslations: Map = {
+type FontWeight = Exclude<TextStyle['fontWeight'], void>;
+
+const FontWeightTranslations: Record<FontWeight, string> = {
```

Comment:
Strict type better that just a `string`

![image](https://user-images.githubusercontent.com/4661784/139598215-7202937f-457b-4d23-baaf-1c76229699c4.png)